### PR TITLE
fixing sanity issues

### PR DESCRIPTION
**Pull request by ar-india**
Signed-off-by: ar-india <alokranjan2005@gmail.com>

### CODE REVIEW

**Comment by ar-india** [comment]
fixing bug 
https://nimblejira.nimblestorage.com/browse/SA-148

File: ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py [hunk 1]
```diff
@@ -1,300 +1,300 @@
...
-
-def get_initiator_group_id(client_obj, ig_name):
-    if is_null_or_empty(ig_name):
-        return None
-    else:
-        resp = client_obj.initiator_groups.get(name=ig_name)
-        if resp is None:
-            raise Exception(f"Invalid value for initiator group {ig_name}")
-        return resp.attrs.get("id")
+#!/usr/bin/python
```
Comment (ar-india) [inline ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py]: no code has been changed in this file. i have just replaced '\r\n\' with \n

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py [hunk 2]
```diff
@@ -673,11 +673,11 @@ def main():
...
-            and (demote is None or demote is False)
-            and (abort_handover is None or abort_handover is False)
-            and (handover is None or handover is False)
-            and (validate is None or validate is False)
-            and (state == "create" or state == "present")):
+              and (demote is None or demote is False)
+              and (abort_handover is None or abort_handover is False)
+              and (handover is None or handover is False)
+              and (validate is None or validate is False)
+              and (state == "create" or state == "present")):
```
Comment (rgcostea) [inline ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py:680]: Are those 2 extra spaces intentional?

**Review by rgcostea — APPROVED** [review]

**Review by suneeth51 — APPROVED** [review]
lgtm..

File: ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py (hunk 1 above)
Comment (datamattsson) [inline ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py]: Please change this to:
```
#!/usr/bin/env python
```
as per https://docs.ansible.com/ansible/2.9/dev_guide/testing/sanity/shebang.html

**Review by datamattsson — CHANGES_REQUESTED** [review]
Compiles clean on supported versions with the shebang change.

File: ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py (hunk 2 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py:680]: yes

File: ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py (hunk 1 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py]: thanks.. it worked.

**Pull request merged**